apps/backend/src/modules/users/user.repository.ts

```diff
@@ -154,6 +190,51 @@ class UserRepository implements Repository {
 			updatedAt: user.updatedAt,
 		});
 	}
+
+	public async updateUserDetails(
```

Comment:
method update also updates only UserDetails model. Why do we need both of this methods?